File: backend/MatriculaServers.Core/Disciplina.cs
namespace MatriculaServers.Core
{
    public class Disciplina : Entity
    {
        public string Nome { get; set; }

        public Disciplina()
        {
        }

        public Disciplina(int id, string nome)
        {
            Id = id;
            Nome = nome;
        }

        public static Disciplina Create(int id, string nome)
            => new Disciplina(id, nome);
    }
}

File: old/src/entities/disciplina.ts
export class Disciplina {
  public Id: string;
  public Nome: string;

  public static Create(id: string, nome: string): Disciplina {
    var disciplina = new Disciplina();
    disciplina.Id = id;
    disciplina.Nome = nome;
    return disciplina;
  }
}
